docs/documentation/Realtime/recent additions.md

```diff
@@ -2,22 +2,7 @@
 
 The GTFS Realtime Specification is not set in stone. Instead, it is an open specification developed and maintained by the community of transit agencies, developers, and other stakeholders who use GTFS Realtime. It is expected that this community of producers and consumers of GTFS Realtime data will have proposals for extending the spec to enable new capabilities.
```

Comment:
Nit: I tend to go for GTFS Schedule/Realtime _**Reference**_ (instead of "specification") because the S in GTFS already refers to "Specification".